Marlin/temperature.cpp

```diff
@@ -1736,7 +1744,16 @@ void Temperature::isr() {
         #define GE0 >=
       #endif
       if (current_temperature_raw[0] GE0 maxttemp_raw[0]) max_temp_error(0);
-      if (minttemp_raw[0] GE0 current_temperature_raw[0]) min_temp_error(0);
+      if (minttemp_raw[0] GE0 current_temperature_raw[0] && !is_preheating(0) && target_temperature[0] > 0.0f) {
```

Comment:
It's a problem/bug that arise with the use of thermistors that don't offer good reading (or any reading at all) at room temperature.

**is_preheating** is like a "grace period" where you're allowed to blindly heat stuff for a while because you know you should soon be able to get decent readings; but if you're still not reading anything after that grace period, then normal security kicks in.

However, when you just turn on your printer at room temperature, you're not trying to heat anything (so is_preheating will return false), but `minttemps_raw[#] GE# current_temperature_raw[#]` will return true, because our thermistor can't read anything below 40°C and everything is turned off before the print can even start. The idea behind `target_temperature[0] > 0.0f ` is "If I'm not actually trying to heat anything, don't throw a min_temp_error"